pom.xml

```diff
@@ -70,6 +72,25 @@
             <artifactId>wildfly-common</artifactId>
             <version>${version.org.wildfly.common.wildfly-common}</version>
         </dependency>
+
+        <dependency>
+            <groupId>javax.json</groupId>
+            <artifactId>javax.json-api</artifactId>
+            <version>${version.javax.json}</version>
+            <!-- only required for the JsonFormatter -->
+            <optional>true</optional>
+        </dependency>
+        <!-- JSON implementation -->
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>javax.json</artifactId>
+            <version>${version.org.glassfish.javax.json}</version>
+            <!-- Marked as provided as any valid JSON implementation can be used if the JsonFormatter is desired -->
+            <scope>provided</scope>
```

Comment:
Should this just marked as `<scope>test</scope>` or possibly even `<scope>compile</scope>` since the `JsonFormatter` wouldn't work with some kind of implementation.